contents/docs/cdp/sources/_snippets/source-bing-ads.mdx

```diff
@@ -0,0 +1,30 @@
+You can sync data from Bing Ads reports by configuring it as a source in PostHog. The supported reports that can be synced include Campaigns, Campaign Performance Report, Ad Group Performance Report, and Ad Performance Report, as described here:
+
+- [Campaigns](https://learn.microsoft.com/en-us/advertising/guides/campaign-management-guides?view=bingads-13)
+- [Campaign Performance Report](https://learn.microsoft.com/en-us/advertising/guides/reports): Performance metrics at the campaign level
+- [Ad Group Performance Report](https://learn.microsoft.com/en-us/advertising/guides/reports): Performance metrics at the ad group level
+- [Ad Performance Report](https://learn.microsoft.com/en-us/advertising/guides/reports): Performance metrics at the ad level
```

Comment:
| Report Type | Description |
|------------|-------------|
| [Campaigns](https://learn.microsoft.com/en-us/advertising/guides/campaign-management-guides?view=bingads-13) | |
| [Campaign Performance Report](https://learn.microsoft.com/en-us/advertising/guides/reports) | Performance metrics at the campaign level |
| [Ad Group Performance Report](https://learn.microsoft.com/en-us/advertising/guides/reports) | Performance metrics at the ad group level |
| [Ad Performance Report](https://learn.microsoft.com/en-us/advertising/guides/reports) | Performance metrics at the ad level |